src/components/tabs/Tabs.astro

```diff
@@ -47,8 +47,6 @@ export interface Props {
 						index
 					);
 				});
-
-				starlightTab.scrollIntoView({ behavior: 'smooth' });
```

Comment:
Thanks for the PR @yasnbouz — I don’t think this is quite what was meant in #5711. This change prevents all scrolling when changing tabs, but this results in a bad user experience with tab content jumping up and down on the page when tabs are toggled.

For example, try switching between the tabs in the ‘Using “atoms”’ section of the sharing state guide: https://docs-git-fork-yasnbouz-fix-tabs-astrodotbuild.vercel.app/en/core-concepts/sharing-state/#using-atoms

When I switch from Preact to Svelte for example, the tabbed content jumps down the page.

Reply:
Thank you, @sarah11918 and @delucis, for your feedback. I apologize for misunderstanding the issue